SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -373,6 +373,43 @@ void USpatialSender::ProcessUpdatesQueuedUntilAuthority(Worker_EntityId EntityId
 	}
 }
 
+void USpatialSender::FlushPackedUnreliableRPCs()
+{
+	if (UnreliableRPCs.Num() == 0)
+	{
+		return;
+	}
+
+	check(NetDriver->GetWorld() != nullptr);
+
+	for (const auto& It : UnreliableRPCs)
```

Comment:
nit: If we're concerned about the additional bloat of the entity id when single events, can we check for if we only have a single pending rpc, and if so use to single rpc route?

Reply:
Adding this might not be a small change, so I'll create a ticket for now.

Reply:
https://improbableio.atlassian.net/browse/UNR-1563